docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3-slim
```

Comment:
Do this need the control machine connected and has access to Docker hub during run as well?